nrt/data/DynaFlow/WeightedDifference/WD.crv

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This nrt can be deleted (keep it so that we put it in the private repo if necessary)